migration.md

```diff
@@ -0,0 +1,89 @@
+# Migration guide
+
+This page documents key considerations when migrating an existing application using legacy Hyperledger Fabric client SDKs to the Fabric Gateway client API.
```

Comment:
- suggest defining what "migrating" means in this context - rewriting, or converting, etc. - and is this "migration" manual, or automated to some extent? 
- your verb "using" -- this doesn't apply to the migration action right, it applies to the writing of the original? i.e. is it "an existing application that was written using the legacy ..."